scraper/src/tests/config_loader/sitemap_test.py

```diff
@@ -0,0 +1,31 @@
+# coding: utf-8
+import os
+
+from ...config.config_loader import ConfigLoader
+from .abstract import config
+import pytest
+
+
+class TestSitemap:
+    def test_need_sitemap_urls_force_sitemap(self):
+        """ Should throw if CONFIG need sitemap_urls """
+        # Given
+
+        urls_sitemap_less = config({'force_sitemap_urls_crawling': True})
+
+        # When / Then
+        with pytest.raises(Exception):
+            ConfigLoader(urls_sitemap_less)
+
+
+    def test_need_sitemap_urls_regex(self):
+        """ Should throw if CONFIG need sitemap_urls """
+        urls_sitemap_less = {
+            'urls_sitemap_regexs': ["blabla"]
```

Comment:
We can change blabla to a real sitemap url 😄 . We can take our doc one

Reply:
I will but /doc/ in order to highlight it is a regex that match documentation url and not a proper url